packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -656,6 +657,18 @@ export interface ServiceProps {
    */
   readonly autoDeploymentsEnabled?: boolean;
 
+  /**
+   * Specifies an App Runner Auto Scaling Configuration.
+   *
+   * A default configuration is either the AWS recommended configuration,
+   * or the configuration you set as the default.
+   *
+   * @see https://docs.aws.amazon.com/apprunner/latest/dg/manage-autoscaling.html
+   *
+   * @default - the latest revision of a default auto scaling configuration is used.
+   */
+  readonly autoScalingConfiguration?: IAutoScalingConfiguration;
```

Comment:
I would prefer just `autoScaling?: IAutoScalingConfiguration;`

Reply:
Thank you. I will keep it as is, in the form of aligning with the class names or property names of the L1 construct.